src/main/java/com/gluonhq/substrate/util/ios/CodeSigning.java

```diff
@@ -93,6 +92,9 @@ public CodeSigning(ProcessPaths paths, InternalProjectConfiguration projectConfi
 
         appPath = paths.getAppPath().resolve(projectConfiguration.getAppName() + ".app");
         tmpPath = paths.getTmpPath();
+
+        providedIdentityName = projectConfiguration.getIosSigningConfiguration().getProvidedSigningIdentity();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4769192)